core/src/test/scala/org/broadinstitute/dsde/rawls/mock/RemoteServicesMockServer.scala

```diff
@@ -331,7 +331,7 @@ class RemoteServicesMockServer(port:Int) extends RawlsTestUtils {
       response()
         .withHeaders(jsonHeader)
         .withBody {
-          (1 to numWorkflows).map(_ => ExecutionServiceStatus(UUID.randomUUID().toString, "Submitted")).toList.toJson.toString
+          (1 to numWorkflows).map(n => ExecutionServiceStatus(UUID.randomUUID().toString, if (n == numWorkflows) "Failed" else "Submitted")).toList.toJson.toString
```

Comment:
hoo boy.  :)

Could you add `// last one fails` or something to make the intention clear?